packages/database/src/migrations/20210527034457-StriveLarvalHabitatBySpeciesBarChart-modifies-data.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_GROUP_CODE = 'PG_Strive_PNG_Country';
+
+const SPECIES = [
+  'Ae. aegypti',
+  'Ae. albopictus',
+  'Ae. scutellaris',
+  'Aedes Other',
+  'Aedes spp.',
+  'An. farauti',
+  'An. koliensis',
+  'An. longirostris',
+  'An. punctulatus',
+  'Culex spp.',
+  'Cx. annulirostris',
+  'Cx. quinquefasciatus',
+  'Cx. sitiens',
+  'Mansoni spp.',
+];
+
+const COLOURS = [
+  '#e22709',
+  '#0df3e4',
+  '#fa6131',
+  '#9bbf87',
+  '#3fc399',
+  '#28d530',
+  '#4549a6',
+  '#c3d146',
+  '#67a6d1',
+  '#4e9469',
+  '#e617d4',
+  '#d84876',
+  '#4744cb',
+  '#b89fbd',
+];
+
+const BASE_DASHBOARD_REPORT = {
+  id: 'PG_Strive_Habitat_By_Species',
+  dataBuilder: 'reportServer',
+  dataServices: [
+    {
+      isDataRegional: true,
+    },
+  ],
+};
+
+const DATA_BUILDER_CONFIG = {
+  reportCode: 'STRIVE_Habitat_By_Species_LHS',
```

Comment:
We still haven't come up with any naming convention for reportCodes and dashboardIds. Ideally they'd follow some format that would make it really easy to look up one based on the other, eg. 
`dashboardId: PG_Strive_Habitat_By_Species_Stacked_Bar_Chart, reportCode: PG_Strive_Habitat_By_Species`

Might be worth something us discussing as a team, but no biggie for now 😄

Reply:
Yeah that is a good point, I followed the previous report codes naming formate, which has no country code, but it seems like they don't have country code because they are used in multiple countries.

Reply:
I still incline to use a shorter `dashboardId` because I prefer a shorter URL, which is clean, clear and more attractive when seen or copied in full, but it is worth to debate in dev chat 😆